analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output-with-nuspec.yml

```diff
@@ -24,11 +24,45 @@ project:
   scopes:
```

Comment:
Just a bit confused that the last commit in this PR is a demonstration of issues. But looking forward to the follow-up PR with the fix.